src/pages/patientView/trialMatch/TrialMatchTable.tsx

```diff
@@ -0,0 +1,354 @@
+                                                                            <li>{genomicMatch.trueHugoSymbol + ' ' + genomicMatch.trueProteinChange + ' '}
+                                                                                {genomicMatch.sampleIds.map((sampleId: string) => (
+                                                                                    <span className={styles.genomicSpan}>
+                                                                                        {props.sampleManager!.getComponentForSample(sampleId, 1, '')}
+                                                                                    </span>
+                                                                                ))}
+                                                                            </li>
+                                                                        ))}
+                                                                        </ul>
+                                                                    </Then>
+                                                                    <Else>
+                                                                        <span>{genomicGroupMatch.matches[0].trueHugoSymbol + ' ' + genomicGroupMatch.matches[0].trueProteinChange + ' '}
+                                                                            {genomicGroupMatch.matches[0].sampleIds.map((sampleId: string) => (
+                                                                                <span className={styles.genomicSpan}>
+                                                                                    {props.sampleManager!.getComponentForSample(sampleId, 1, '')}
+                                                                                </span>
+                                                                            ))}
+                                                                        </span>
+                                                                    </Else>
+                                                                </If>
+                                                            </span>
+                                                        </div>
+                                                    </Else>
+                                                </If>
+                                            ))}
+                                            <If condition={clinicalGroupMatch.notMatches.length > 0}>
+                                                <DefaultTooltip
+                                                    placement='bottomLeft'
+                                                    trigger={['hover', 'focus']}
+                                                    overlay={this.tooltipGenomicContent(clinicalGroupMatch.notMatches)}
+                                                    arrowContent={<div className="rc-tooltip-arrow-inner" />}
```

Comment:
Don't specify arrowContent if you don't want to change the default.

Reply:
Removed arrowContent